c#-mini/.md

```diff
@@ -0,0 +1,142 @@
+# C# : A minimalistic approach
+
+I'm a C# enthusiast, but I'm also an all-around minimalist. Which sounds a little bit paradoxical, knowing that C# has never been a minimalistic language. Which also means it is not considered good for quick prototyping or scripting. Gladly, there's a shift in the latest language versions. In this article, I'll cover what has already changed and what can be done in the future to make C# even more minimalistic.
```

Comment:
After scripting add:

I think that is one of the main reasons why it's lacking behind python in the popularity contest.

And then start a new paragraph